pages.pt-BR/common/wpa_supplicant.md

```diff
@@ -0,0 +1,11 @@
+# wpa_supplicant
+
+> Gerenciador de redes wireless protegidas.
+
+- Entrar em uma rede wireless protegida:
+
+`wpa_supplicant -i {{interface}} -c {{caminho/para/wpa_supplicant_conf.conf}}`
+
+- Entrar emuma rede wireless protegita e executar em um daemon:
```

Comment:
Uh-oh, a typo sneaked in here ;) I'll submit a follow-up PR to fix.

Reply:
Yeah,
Nice that you noticed it.

Reply:
As they say... "Many eyes make bugs shallow" :heart:

Reply:
By the way, the update PR was #2889.